examples/fetch/index.tsx

```diff
@@ -23,11 +23,55 @@ const App = () => {
     }
   };
 
+  async function streamToArrayBuffer(stream: any) {
+    const reader = stream.getReader();
+    while (true) {
+      console.log("await read");
+      const { done, value } = await reader.read();
+      console.log("done", done);
+      if (done) {
+        break;
+      } else {
+        console.log(value.byteLength);
+        // @ts-ignore
+        const text = globalThis.TextCodecHelper.decode(value);
+        setData(text);
+        const currentTime = new Date();
+        console.log(text);
+        // @ts-ignore
+        console.log("duration ", currentTime.getTime() - globalThis.startTime.getTime());
+      }
+    }
+  }
+
+  const handleButtonClickStreaming = async () => {
+    setError(null);
+
+    try {
+      console.log("fetch streaming");
+      // @ts-ignore
+      globalThis.startTime = new Date();
+      // @ts-ignore
+      lynx.fetch("https://e3e0932a-9522-4aec-aec9-12cba94b0c7f.mock.pstmn.io/chunk", {
+        // @ts-ignore
```

Comment:
Instead of `ts-ignore`, can we provide correct types in `@lynx-js/types` for Lynx-extended params?